### PR TITLE
:robot: Add vscode.vm

### PR DESCRIPTION
**Pull request by vm-packages**
Automated PR from https://github.com/mandiant/VM-Packages/issues/887 :rocket:

### CODE REVIEW

File: packages/vscode.vm/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -0,0 +1,15 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+  $toolName = 'VSCode'
+  $category = 'Text Editors'
```
Comment (Ana06) [inline packages/vscode.vm/tools/chocolateyinstall.ps1:6]: We are not adding other Text Editors (like notepad++) to the Tools folder. I have no strong opinion if we should add all the Text Editors or not, but we should do this in a consistent way. @vm-packages opinions?

Related: https://github.com/mandiant/VM-Packages/issues/883

Comment (mr-tz) [reply packages/vscode.vm/tools/chocolateyinstall.ps1:6]: agreed, no strong feelings either way - so let's do what is easiest to maintain...

**Review by mr-tz — APPROVED** [review]
LGTM besides the comment

File: packages/vscode.vm/tools/chocolateyinstall.ps1 (hunk 1 above)
Comment (emtuls) [reply packages/vscode.vm/tools/chocolateyinstall.ps1:6]: I'm all for throwing this into the `productivity tools` category of we go that route. 🙂

**Comment by RexChenjq** [comment]
Hi team, just noticed it today that VScode installation directory is under `ProgramFiles` which doesn't survive sysprep. I noticed most of the other tools are installed under `${Env:ChocolateyInstall}` which does survive sysprep.
Is there any plan to achieve some level of consistency for installation location? 

Cheers!

**Pull request merged**